vm/src/builtins/object.rs

```diff
@@ -308,7 +308,7 @@ impl PyBaseObject {
 
     /// Return hash(self).
     #[pymethod(magic)]
-    fn hash(zelf: PyObjectRef, vm: &VirtualMachine) -> PyResult<PyHash> {
+    pub fn hash(zelf: PyObjectRef, vm: &VirtualMachine) -> PyResult<PyHash> {
```

Comment:
where is this used? I think `vm.hash()` is commonly used but not about this one. So I want to know where is this used.

Reply:
Originally, I tried to use it for the `Hashable for PyGenericAlias` implementation but it is using `vm._hash` as you said. I'll drop the commit. Thanks!